.tern-project

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Please revert this file as it's an unrelated change.

Reply:
Hello, 

How can I do to revert this file ?